lib/index.js

```diff
@@ -68,6 +70,22 @@ exports.global = window;
 
 Segment.prototype.initialize = function() {
   var self = this;
+
+  if (this.options.retryQueue) {
+    this._lsqueue = new Queue('segmentio', function(item, done) {
+      // Update the sentAt time each retry so the tracking-api doesn't interperet a time skew
+      item.sentAt = new Date();
+      // send
+      send(item.url, item.msg, item.headers, function(err, res) {
```

Comment:
We actually explicitly don't want to batch for a handful of reasons, mainly its harder to debug, and batching requires a forced delay.